src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,11 +10,14 @@
  */
 public class Person implements ReadOnlyPerson {
 
+	private static int nextSequenceNumber = 0; 
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)